blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
```

Comment:
I thought "reviewed" corresponded to state based on the events (such as a response seen stamp), rather than just being UI state. Although if we're persisting the reviewed state, then I think it still requires a bit of thought (since selected is purely client-side, while the notion of reviewed is part of the wider data model, and therefore somewhat distinct).